applications/workers/workers-smoketest/src/smokeTest/kotlin/net/corda/applications/workers/smoketest/ledger/ConsensualLedgerTest.kt

```diff
@@ -0,0 +1,137 @@
+package net.corda.applications.workers.smoketest.ledger
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import net.corda.applications.workers.smoketest.RPC_FLOW_STATUS_FAILED
+import net.corda.applications.workers.smoketest.RPC_FLOW_STATUS_SUCCESS
+import net.corda.applications.workers.smoketest.awaitRpcFlowFinished
+import net.corda.applications.workers.smoketest.conditionallyUploadCordaPackage
+import net.corda.applications.workers.smoketest.getHoldingIdShortHash
+import net.corda.applications.workers.smoketest.getOrCreateVirtualNodeFor
+import net.corda.applications.workers.smoketest.registerMember
+import net.corda.applications.workers.smoketest.startRpcFlow
+import net.corda.v5.crypto.SecureHash
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS
+
+/**
+ * This file uses a dedicated CPI, Group Id and X500 names from ConsensualLedgerConfig.
```

Comment:
```suggestion
 * This file uses a dedicated CPI, Group Id and X500 names.
```
? Or maybe remove it completely?